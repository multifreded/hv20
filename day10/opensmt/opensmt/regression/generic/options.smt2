(set-option :print-success true)
(set-option :foo bar)
(get-option :foo)
(get-option :print-success)
; should fail
(set-option :print-success quux)
