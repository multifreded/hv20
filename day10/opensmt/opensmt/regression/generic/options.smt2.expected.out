Syntax error: expecting either 'true' or 'false', got 'quux'
